tests/forms_tests/tests/test_forms.py

```diff
@@ -2887,6 +2887,36 @@ class DateAgeForm(Form):
         form = DateAgeForm(data, initial={"date_age": ["200-10-10", 14]})
         self.assertTrue(form.has_changed())
 
+    def test_clean_disabled_multivalue(self):
```

Comment:
The test would be better in `tests/forms_tests/field_tests/test_multivaluefield.py`. Maybe you can reuse one of the existing `MultiValueField` subclasses from there rather than adding another one in the test.